src/Register.jsx

```diff
@@ -1,5 +1,37 @@
-export const Register = () => {
+import React, { useState } from "react"
+import { Welcome } from './Welcome'
+
+export const Register = (props) => {
+    const [email, setEmail] = useState('');
+    const [pass, setPass] = useState('');
+    const [name, setName] = useState('');
+    const[page, setPage] = useState(false)
+
+
+    const handleSubmit = (e) => {
+        e.preventDefault(); // Prevent default form submission behavior
+        setPage(true);
+    }
+
     return (
-        <>Register</>
+        <div>
+            {!page &&
+                <div className="auth-form-container"> 
+                    <h2> Register </h2>  
+                    <form className="register-form" onSubmit={handleSubmit}>
+                        <label>Full Name </label>
+                        <input value={name} name="name" id="id" placeholder="full name" />
+                        <label htmlFor="email">Email </label>
+                        <input value={email} type="email" placeholder="youremail@gmail.com" name="email" />
+                        <label htmlFor="password">Password </label>
+                        <input value={pass} type="password" placeholder="*****" name="password" />
+                        <button className='login-btn'>Log In</button>
+                    </form>
+                    <button className='register-btn' onClick={() => props.onFormSwitch('login')}>Login Here </button>
+                </div>
+            }
+
+            {page && <Welcome />}
```

Comment:
make this an if/else, as oppose to the two if's it is now

Reply:
made this change in the code, i believe.